joblib/test/test_memory.py

```diff
@@ -569,8 +558,16 @@ def test_memory_in_memory_function_code_change(tmpdir):
 
 
 def test_clear_memory_with_none_cachedir():
-    mem = Memory(cachedir=None)
-    mem.clear()
+    memory = Memory(cachedir=None)
+    memory.clear()
+
+
+def func_with_kwonly_args(a, b, kw1='kw1', kw2='kw2'):
```

Comment:
Yes I added them because that flake8 job repeatedly gave a red at some point of time. I am puzzled why it doesn't, now. Anyways this is good :+1: